jinja2/filters.py

```diff
@@ -696,8 +696,6 @@ def do_round(value, precision=0, method='common'):
 _GroupTuple = namedtuple('_GroupTuple', ['grouper', 'list'])
 _GroupTuple.__repr__ = tuple.__repr__
 _GroupTuple.__str__ = tuple.__str__
-if not PY2:
```

Comment:
I think so too but it is still not necessarily, unicode(_GroupTuple('test', ('foo', 'bar'))) is still calling the overloaded repr.